### ISSUE TITLE
HDFS connector crashed

### ISSUE THREAD

**Issue by yatharthranjan** [RADAR-CNS/RADAR-HDFS-Sink-Connector#6]
The HDFS connector crashed unexpectedly and keeps restarting and giving the same error. 
```
org.apache.avro.SchemaParseException: Can't redefine: org.radarcns.active.thincit.ThincItCodeBreakerQuarterSummary
	at org.apache.avro.Schema$Names.put(Schema.java:1060)
	at org.apache.avro.Schema$NamedSchema.writeNameRef(Schema.java:509)
	at org.apache.avro.Schema$RecordSchema.toJson(Schema.java:629)
	at org.apache.avro.Schema$RecordSchema.fieldsToJson(Schema.java:651)
	at org.apache.avro.Schema$RecordSchema.toJson(Schema.java:638)
	at org.apache.avro.Schema$RecordSchema.fieldsToJson(Schema.java:651)
	at org.apache.avro.Schema$RecordSchema.toJson(Schema.java:638)
	at org.apache.avro.Schema.toString(Schema.java:297)
	at org.apache.avro.Schema.toString(Schema.java:287)
	at org.apache.avro.file.DataFileWriter.create(DataFileWriter.java:138)
	at org.radarcns.sink.hdfs.AvroRecordWriterProviderRadar.getRecordWriter(AvroRecordWriterProviderRadar.java:89)
	at io.confluent.connect.hdfs.TopicPartitionWriter.getWriter(TopicPartitionWriter.java:458)
	at io.confluent.connect.hdfs.TopicPartitionWriter.writeRecord(TopicPartitionWriter.java:534)
	at io.confluent.connect.hdfs.TopicPartitionWriter.write(TopicPartitionWriter.java:294)
	at io.confluent.connect.hdfs.DataWriter.write(DataWriter.java:234)
	at io.confluent.connect.hdfs.HdfsSinkTask.put(HdfsSinkTask.java:103)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.deliverMessages(WorkerSinkTask.java:384)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.poll(WorkerSinkTask.java:240)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.iteration(WorkerSinkTask.java:172)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.execute(WorkerSinkTask.java:143)
	at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:140)
	at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:175)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[2017-11-13 14:21:50,433] ERROR Task is being killed and will not recover until manually restarted (org.apache.kafka.connect.runtime.WorkerSinkTask)
[2017-11-13 14:21:50,461] INFO WorkerSinkTask{id=radar-hdfs-sink-android-15000-0} Committing offsets (org.apache.kafka.connect.runtime.WorkerSinkTask)
[2017-11-13 14:21:50,495] ERROR Task radar-hdfs-sink-android-15000-0 threw an uncaught and unrecoverable exception (org.apache.kafka.connect.runtime.WorkerTask)
org.apache.kafka.connect.errors.ConnectException: Exiting WorkerSinkTask due to unrecoverable exception.
	at org.apache.kafka.connect.runtime.WorkerSinkTask.deliverMessages(WorkerSinkTask.java:406)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.poll(WorkerSinkTask.java:240)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.iteration(WorkerSinkTask.java:172)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.execute(WorkerSinkTask.java:143)
	at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:140)
	at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:175)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

```
This probably happened when the data from Thinc It app was sent to kafka.

**Comment by blootsvoets** [comment]
This should be fixed with #5, if you use the `AvroTopicPartitioner` partitioner class.

**Comment by yatharthranjan** [comment]
@blootsvoets can we merge the PR and create a new release soon?

**Comment by yatharthranjan** [comment]
Closed by #5
